integration-tests/deployment/ccip/state.go

```diff
@@ -219,7 +213,7 @@ func LoadChainState(chain deployment.Chain, addresses map[string]deployment.Type
 			if err != nil {
 				return state, err
 			}
-			state.RMNRemote = rmnRemote
+			state.RMNRemote = rmn1_6.New(rmnRemote)
```

Comment:
how come `rmn1_6` if this case statement is pointing to `Version1_0_0`?

```
case deployment.NewTypeAndVersion(RMNRemote, deployment.Version1_0_0...
```

Reply:
@ogtownsend can you point me to the code ref ? I see that TypeAndVersion for RMN is [RMNRemote 1.6.0-dev](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/ccip/rmn/RMNRemote.sol#L17)

Reply:
I see what you mean, @connorwstein I think the case statement should be changed to `Version1_6_0_dev`, am I missing something?